host/xdc/event_generator.go

```diff
@@ -0,0 +1,374 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
i would put this in common/xdc

Reply:
and copyright 2019

Reply:
Updated the location and the copyright header